packages/spor-elm/src/Spor/Text.elm

```diff
@@ -76,5 +77,8 @@ withString text (Text options) =
 toHtml : Text -> Html a
 toHtml (Text options) =
     Html.p
-        [ Attribute.css <| TextStyleInternal.toCss options.style ]
+        [ Attribute.css <|
+            TextStyleInternal.toCss options.style
+                ++ [ Css.margin Css.zero ]
```

Comment:
Hva gjør denne?

Reply:
Dette gir en paragraf med margins, men i mitt tilfelle ønsker jeg ikke disse marginene, så jeg nullstiller dem

> Jeg liker navnene du har valgt, de er veldig like som de jeg har valgt for React og React Native – veldig bra!
> 
> Ellers ser jeg at det ikke er støtte for avviks-linjetags – det gjør kanskje ikke noe i første versjon. Si ifra når det skal implementeres da – jeg har noen tanker om navngivning.

Ja, jeg skrev en TODO i beskrivelsen for dette

Reply:
Men vil du nullstille dem for alle Text-komponenter?

Reply:
Det er en stund siden jeg la den endringen nå (så det i forbindelse med testing av taggene), så tror heller det kan være bedre å bruke en additional styles eller lignende som props i komponenten her. En paragraf (Html.p) får automatisk med margins, med mindre man nullstiller dem (og det er jo ikke alltid man ønsker samme type margins)